tests/snappi_tests/multidut/bgp/test_bgp_outbound_interconnectivity_flap.py

```diff
@@ -0,0 +1,94 @@
+import pytest
```

Comment:
change the file/function name to downlink_port_flap

Reply:
implemented